src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
+const HEALTH = 100;
+const DAMAGE_PER_BITE = 50;
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = HEALTH;
+    Animal.alive.push(this);
+  }
```

Comment:
To answer your question from the comment: in this case, you don't need to set a default value via a constant, because that value itself will tell you what it means and it won't be considered a magic value:
```suggestion
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
    Animal.alive.push(this);
  }
```
You also need to be able to pass a different value for the `health` field to the constructor, so it is important to use this option